### PR TITLE
[Docker] Fix broken gpg --verify and Dockerfile cleanup

### LINKED ISSUE #1588: Remove extra layer before key RUN in docker image
ITNOA

**Describe the bug**
As you can see in `docker/master/Dockerfile`, we have two RUN immediately, that I think it is not necessary and making extra layer, so I remove it.

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

As you can see in `docker/master/Dockerfile`, we have two RUN immediately, that I think it is not necessary and making extra layer, so I remove it.

This PR resolved #1588

### CODE REVIEW

**Comment by soroshsabz** [comment]
@andywolk Hi

```
2022-03-20 22:00:00 (59.5 MB/s) - '/usr/local/bin/gosu.asc' saved [543/543]

gpg: assuming signed data in '/usr/local/bin/gosu'
gpg: Signature made Wed Nov 19 07:57:11 2014 UTC
gpg:                using RSA key 036A9C25BF357DD4
gpg: Good signature from "Tianon Gravi <tianon@tianon.xyz>" [unknown]
gpg:                 aka "Andrew Page (Tianon Gravi) <admwiggin@gmail.com>" [unknown]
gpg:                 aka "Tianon Gravi <tianon@debian.org>" [unknown]
gpg:                 aka "Tianon Gravi (Andrew Page) <tianon@infosiftr.com>" [unknown]
gpg:                 aka "Tianon Gravi <tianon@dockerproject.org>" [unknown]
gpg:                 aka "Andrew Page (tianon) <andrew@infosiftr.com>" [unknown]
gpg:                 aka "Andrew Page (tianon) <andrew@vitalroute.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: B42F 6819 007F 00F8 8E36  4FD4 036A 9C25 BF35 7DD4
```

But before this log generated I can see

```
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: key 036A9C25BF357DD4: 11 duplicate signatures removed
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key 036A9C25BF357DD4: public key "Tianon Gravi <tianon@tianon.xyz>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: key 36B4249FA7B0FB03: public key "FreeSWITCH Packaging Key <freeswitch@signalwire.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

How to resolve this? (As you can see in above log, `036A9C25BF357DD4` key is imported

File: docker/master/Dockerfile [hunk 1]
```diff
@@ -10,18 +10,17 @@ ARG TOKEN
...
 
 # grab gosu for easy step-down from root
-RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 \
+RUN apt-get update && apt-get install -y --no-install-recommends dirmngr gnupg2 ca-certificates wget \
     && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
     && gpg2 --keyserver hkp://keyserver.ubuntu.com --recv-keys 655DA1341B5207915210AFE936B4249FA7B0FB03 \
     && gpg2 --output /usr/share/keyrings/signalwire-freeswitch-repo.gpg --export 655DA1341B5207915210AFE936B4249FA7B0FB03 \
-    && apt-get purge -y --auto-remove gnupg2
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates wget && rm -rf /var/lib/apt/lists/* \
+    rm -rf /var/lib/apt/lists/* \
```
Comment (andywolk) [inline docker/master/Dockerfile:17]: Missing &&

**Review by andywolk — CHANGES_REQUESTED** [review]

File: docker/master/Dockerfile (hunk 1 above)
Comment (soroshsabz) [reply docker/master/Dockerfile]: sorry :(

**Review by andywolk — APPROVED** [review]

**Comment by soroshsabz** [comment]
@andywolk Hi

```
2022-03-20 22:00:00 (59.5 MB/s) - '/usr/local/bin/gosu.asc' saved [543/543]

gpg: assuming signed data in '/usr/local/bin/gosu'
gpg: Signature made Wed Nov 19 07:57:11 2014 UTC
gpg:                using RSA key 036A9C25BF357DD4
gpg: Good signature from "Tianon Gravi <tianon@tianon.xyz>" [unknown]
gpg:                 aka "Andrew Page (Tianon Gravi) <admwiggin@gmail.com>" [unknown]
gpg:                 aka "Tianon Gravi <tianon@debian.org>" [unknown]
gpg:                 aka "Tianon Gravi (Andrew Page) <tianon@infosiftr.com>" [unknown]
gpg:                 aka "Tianon Gravi <tianon@dockerproject.org>" [unknown]
gpg:                 aka "Andrew Page (tianon) <andrew@infosiftr.com>" [unknown]
gpg:                 aka "Andrew Page (tianon) <andrew@vitalroute.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: B42F 6819 007F 00F8 8E36  4FD4 036A 9C25 BF35 7DD4
```

But before this log generated I can see

```
gpg: directory '/root/.gnupg' created
gpg: keybox '/root/.gnupg/pubring.kbx' created
gpg: key 036A9C25BF357DD4: 11 duplicate signatures removed
gpg: /root/.gnupg/trustdb.gpg: trustdb created
gpg: key 036A9C25BF357DD4: public key "Tianon Gravi <tianon@tianon.xyz>" imported
gpg: Total number processed: 1
gpg:               imported: 1
gpg: key 36B4249FA7B0FB03: public key "FreeSWITCH Packaging Key <freeswitch@signalwire.com>" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

**How to resolve this? (As you can see in above log, `036A9C25BF357DD4` key is imported**

**Comment by soroshsabz** [comment]
Thanks for merging

**Pull request merged**